versions.tf

```diff
@@ -1,8 +1,12 @@
 
 terraform {
-  required_version = ">= 0.13.2"
+  required_version = ">= 1.0.9"
 
   required_providers {
-    aws = "~> 3.0"
+    aws = {
```

Comment:
@angelofenoglio  So the fix is only this piece here, right?

Reply:
Essentially yes, but all the references need to be updated also.

Reply:
Seems reasonable, I'll make these changes then!

Reply:
Oh, is that right? I would've thought that the following is implicit:
```
resource "foo" "bar" {
    ...
    provider = aws
    ...
}
```
And thus it does not need to be explicitly defined.